src/components/PopupWithForm.js

```diff
@@ -22,6 +25,27 @@ export default class PopupWithForm extends Popup {
     this.close();
   }
 
+  setLoadingState(isLoading, defaultText = "Save") {
```

Comment:
Needs correction: you should change text contents of the button here:

```js
 setLoadingState(isLoading) {
   this._submitButton.textContent = isLoading ? "...Saving" : this._submitButtonText;
}
```

Reply:
corrected